pkg/scheduler/core/generic_scheduler.go

```diff
@@ -90,7 +89,7 @@ func (g *genericScheduler) snapshot() error {
 // Schedule tries to schedule the given pod to one of the nodes in the node list.
 // If it succeeds, it will return the name of the node.
 // If it fails, it will return a FitError error with reasons.
-func (g *genericScheduler) Schedule(ctx context.Context, extenders []framework.Extender, fwk framework.Framework, state *framework.CycleState, pod *v1.Pod) (result ScheduleResult, err error) {
+func (g *genericScheduler) Schedule(percentageOfNodesToScore int32, ctx context.Context, extenders []framework.Extender, fwk framework.Framework, state *framework.CycleState, pod *v1.Pod) (result ScheduleResult, err error) {
```

Comment:
Keep `context` as the first parameter: https://github.com/golang/go/wiki/CodeReviewComments#contexts

Nit: we can shorten `percentageOfNodesToScore` to `pctOfNodesToScore`, but up to you :)